src/createActionArgs.js

```diff
@@ -40,17 +40,37 @@ var createStateArg = function (action, model, isAsync, compute) {
   return state
 }
 
+var createServicesArg = function (action, services, parentKey) {
+  return Object.keys(services).reduce(function (newServices, key) {
+    if (typeof services[key] === 'function') {
+      newServices[key] = function () {
+        action.services.push({
+          name: parentKey || 'root',
+          method: key,
+          args: [].slice.call(arguments)
+        })
+        return services[key].apply(services[key], arguments)
+      }
+    } else {
+      newServices[key] = createServicesArg(action, services[key], key)
```

Comment:
it is breaking change since it breaks support of non-function values in services. Am I right?